VivianDoreen/store-manager-frontend
# chore(CodeClimate):  Run CodeClimate Test Coverage

## What does this PR do?
This PR runs the codeclimate test coverage.

## Description of Task to be completed?
The task involves creating a .circleci directory to which a config.yml is added.
Furthermore, the circle ci is connected to the repo.

## How should this be manually tested?
Deploy your branch to the repo and check out the link below:
https://circleci.com/gh/VivianDoreen/store-manager-frontend

## What are the relevant pivotal tracker stories?
[#164329024](https://www.pivotaltracker.com/story/show/164329024)
